pgrx-pg-sys/build.rs

```diff
@@ -88,24 +88,22 @@ impl bindgen::callbacks::ParseCallbacks for PgrxOverrides {
 }
 
 fn main() -> eyre::Result<()> {
-    if std::env::var("DOCS_RS").unwrap_or("false".into()) == "1" {
+    if env_tracked("DOCS_RS").unwrap_or("false".into()) == "1" {
         return Ok(());
     }
 
     // dump the environment for debugging if asked
-    if std::env::var("PGRX_BUILD_VERBOSE").unwrap_or("false".to_string()) == "true" {
+    if env_tracked("PGRX_BUILD_VERBOSE").unwrap_or_else(|| "false".to_string()) == "true" {
         for (k, v) in std::env::vars() {
             eprintln!("{}={}", k, v);
         }
     }
 
     let compile_cshim =
-        std::env::var("CARGO_FEATURE_CSHIM").unwrap_or_else(|_| "0".to_string()) == "1";
+        env_tracked("CARGO_FEATURE_CSHIM").unwrap_or_else(|| "0".to_string()) == "1";
 
-    let is_for_release = std::env::var("PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE")
-        .unwrap_or("0".to_string())
-        == "1";
-    println!("cargo:rerun-if-env-changed=PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE");
+    let is_for_release =
+        env_tracked("PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE").unwrap_or("0".to_string()) == "1";
```

Comment:
Trivial, I know, and preexisting, but it bugs me these aren't consistent.
```suggestion
        env_tracked("PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE").unwrap_or_else(|| "0".to_string()) == "1";
```

Reply:
Also looking again this should actually be another case of `.as_ref() == Some("1")`